### DISCUSSION TITLE
日美擬限制半導體、人工智慧等技術出口

### DISCUSSION THREAD

**Discussion by kaoyihua27** [1101-GS3073-C/Python-Community-GS3073-C#199] [General]
https://www.chinatimes.com/realtimenews/20220110001782-260409?chdtv
日本《讀賣新聞》今(10)日援引多名消息人士稱，日本和美國政府正在共同制定一個框架協議，以限制有關國家安全的敏感技術的出口。

**Comment by jim11261314** [comment]
AI的技術如果被極端國家拿到，是否會產生西班牙入侵南美洲的歷史悲劇
也就是技術上嚴重不對等的侵略行為
